web/index.html

```diff
@@ -58,5 +78,31 @@
 <body>
     <div style="position: absolute; top: 0; left: 0; right: 0; height: 30px;" id="drag-area"></div>
     <div id="root"></div>
+    <div id="splash" style="position: absolute; top: 0; left: 0; right: 0; background-color: white; width: 100%; height: 100%; display: flex; justify-content: center; align-items: center;">
+        <div class="splash-logo">
+            <svg fill="none" width="100%" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
+                <path d="M34.0003 73H66.0003V63.5H45.7003V54.6H63.0003V44.9H45.7003V36.4H66.0003V27H34.0003V73Z" fill="#0A1B34"/>
+                <path d="M90.0003 50C90.0003 59.6 86.6003 68.4 81.0003 75.3L88.1003 82.4C95.5003 73.7 100 62.4 100 50C100 37.8 95.6003 26.6 88.4003 18L81.3003 25.1C86.7003 31.9 90.0003 40.6 90.0003 50Z" fill="#03D47C"/>
+                <path d="M75.3003 81C68.4003 86.6 59.6003 90 50.0003 90C40.6003 90 31.9003 86.7 25.1003 81.3L18.0003 88.4C26.7003 95.7 37.9003 100 50.0003 100C62.4003 100 73.7003 95.5 82.4003 88.1L75.3003 81Z" fill="#0185FF"/>
+                <path d="M19.3 75.7C13.5 68.7 10 59.8 10 50C10 27.9 27.9 10 50 10C59.8 10 68.7 13.5 75.7 19.3L82.8 12.2C74 4.6 62.5 0 50 0C22.4 0 0 22.4 0 50C0 62.5 4.6 74 12.2 82.8L19.3 75.7Z" fill="#FED607"/>
+            </svg>
+         </div>
+    </div>
+    <script>
```

Comment:
Is there any reason we shouldn't:

1. Move this JS code to another file, `web/index.js`
1. Then move this script tag to the header w/ the `defer` attribute

I think that gives us a few benefits:

- The HTML and JS are split into smaller files and downloaded in parallel, which _should_ make them both available faster than if they're part of the same larger file.
- The JS code can be separated out
- The script tag can live in the header instead of the bottom of the file (not critical, but nice for html organization)

Reply:
I think those optimizations are fair in general but for the splash use case I think we need this small script inlined so browser does not need to fetch since we need to run this script as soon as we can on bootstrap